src/test/java/fi/otavanopisto/kuntaapi/test/server/integration/linkedevents/LinkedEventsTestsIT.java

```diff
@@ -0,0 +1,129 @@
+package fi.otavanopisto.kuntaapi.test.server.integration.linkedevents;
+
+import static com.jayway.restassured.RestAssured.given;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsNull.notNullValue;
+import static org.hamcrest.core.IsNull.nullValue;
+import static org.junit.Assert.assertEquals;
+
+import java.time.ZoneId;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+import com.github.tomakehurst.wiremock.junit.WireMockRule;
+import com.jayway.restassured.http.ContentType;
+
+import fi.otavanopisto.kuntaapi.server.integrations.linkedevents.LinkedEventsConsts;
+import fi.otavanopisto.kuntaapi.test.AbstractIntegrationTest;
+
+@SuppressWarnings ("squid:S1192")
+public class LinkedEventsTestsIT extends AbstractIntegrationTest {
+
+  private static final ZoneId TIMEZONE_ID = ZoneId.of("UTC");
+  
+  /**
+   * Starts WireMock
+   */
+  @Rule
+  public WireMockRule wireMockRule = new WireMockRule(getWireMockPort());
+  
+  @Before
+  public void beforeTest() throws InterruptedException {
+    getPtvOrganizationMocker()
+      .mock("9355a207-efd3-4cfb-a02b-67187f34c822");
+    
+    getLinkedEventsEventMocker()
+      .mockEvents("mantyharju:pm-maastojuoksukilpailut-la-27","mantyharju:tanssit-la-872017-klo-2000","mantyharju:venetsialaiset-la-2682017-klo-");
+    
+    startMocks();
+
+    waitApiListCount("/organizations", 1);
+    
+    createLinkedEventsSettings(getOrganizationId(0));
+
+    waitApiListCount(String.format("/organizations/%s/events", getOrganizationId(0)), 3); 
+  }
+
+  @After
+  public void afterClass() {
+    String organizationId = getOrganizationId(0);
+    deleteLinkedEventsSettings(organizationId);
+  }
+  
+  @Test
+  public void testFindEvents() {
+    String organizationId = getOrganizationId(0);
+    
+    System.out.println(getOrganizationEventId(organizationId, 0));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)